### PR TITLE
refactor/gui-v2-settings-modal-added

### PR DESCRIPTION
**Pull request by mustafapc19**
Reference issue: #2807 

## Change Summary

Added settings modal

- Currently all tabs are `AuditTab`, will integrate the rest.
- Grey-ified scrollbar of the modal.

Audit Tab changes:

- Removed `loadProject` logic and removed `projectId` prop from Audit tab.
- Moved pagination to the top to better accommodate scroll.
- Grey-ified table header, reload button.

UI:
image

### CODE REVIEW

File: packages/nc-gui-v2/components/dashboard/settings/SettingsModal.vue [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+  </a-modal>
+</template>
+
+<style scoped>
+.modal-body {
+  height: 70vh;
+}
+::-webkit-scrollbar-thumb {
+  background-color: darkgrey;
+}
```
Comment (bcakmakoglu) [inline packages/nc-gui-v2/components/dashboard/settings/SettingsModal.vue:138]: replace this with WindiCSS rules.

```css
.modal-body {
	@apply h-[70vh] scrollbar-thumb-gray-500;
}
```

File: packages/nc-gui-v2/components/dashboard/settings/AuditTab.vue [hunk 2]
```diff
@@ -45,35 +39,36 @@ async function loadAudits(page = currentPage, limit = currentLimit) {
 
 onMounted(async () => {
   if (audits === null) {
-    await loadProject(projectId)
     await loadAudits(currentPage, currentLimit)
   }
 })
 
+const tableHeaderRenderer = (label: string) => (value: { text: string }) => h('div', { class: 'text-gray-500' }, label)
```
Comment (bcakmakoglu) [inline packages/nc-gui-v2/components/dashboard/settings/AuditTab.vue:46]: unused `value`? could be removed I guess

File: packages/nc-gui-v2/components/dashboard/settings/SettingsModal.vue (hunk 1 above)
Comment (mustafapc19) [reply packages/nc-gui-v2/components/dashboard/settings/SettingsModal.vue:138]: @bcakmakoglu The scrollbar styles doesn't seem to apply in this way.

Comment (mustafapc19) [reply packages/nc-gui-v2/components/dashboard/settings/SettingsModal.vue]: Made it work. Had to apply the scroll style to its child div.

**Comment by o1lab** [comment]
if its good - would love to get this in ?

**Review by bcakmakoglu — APPROVED** [review]

**Pull request merged**